src/Mod/TechDraw/Gui/TaskCenterLine.ui

```diff
@@ -371,20 +371,6 @@
      </item>
     </layout>
    </item>
-   <item>
-    <widget class="QCheckBox" name="cbFlip">
-     <property name="enabled">
-      <bool>false</bool>
-     </property>
-     <property name="toolTip">
-      <string>Flips endpoints of selected lines for centerline creation,
```

Comment:
This tooltip text is deleted. Is it substituted somewhere else then?

Reply:
> This tooltip text is deleted.

Yes, since the checkbox that it described was deleted too.